app/assets/javascripts/controllers/tag_groups/controller.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ColorPick from 'components/tag_groups/ColorPick';
+
+export default () => {
+  const bgColor = $('[name=bg-color]').data('current-color') || "#F17013";
+
+  ReactDOM.render(<ColorPick color ={ bgColor }/>, document.getElementById('tag_group_color'));
+
+  $(document).ready(function(){
+    $(document).bind('ajaxError', function(event, jqxhr, settings, exception){
+      $("#new_tag_group").render_form_errors( $.parseJSON(jqxhr.responseText) );
+    });
+  });
+
+  (function($) {
+
+    $.fn.modal_success = function(){
```

Comment:
This function is called every time the ajax request succed.
Check that:
https://github.com/Codeminer42/cm42-central/blob/cc88e955fc8c7231d0889fd25594f87e80add41f/app/views/tag_groups/show.js.erb#L2

PS : Changed to camelCase.